src/Clock.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Clock extends React.Component {
+  state = {
+    date: new Date(),
+  }
+
+  time = setInterval(() => {
+    this.setState({ date: new Date() });
+
+    // eslint-disable-next-line no-console
+    console.log(this.state.date.toLocaleTimeString());
+  }, 1000);
```

Comment:
you should set interval in ComponentDidMount method

Reply:
https://reactjs.org/docs/state-and-lifecycle.html#adding-lifecycle-methods-to-a-class